### PR TITLE
Retrocompat Bragi2-Bragi7: fix city filter

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

File: libs/mimir/src/adapters/primary/bragi/api.rs [hunk 1]
```diff
@@ -249,6 +253,7 @@ impl Type {
             Type::StopArea => Stop::static_doc_type(),
             Type::Street => Street::static_doc_type(),
             Type::Zone => Admin::static_doc_type(),
+            Type::City => Admin::static_doc_type(),
```
Comment (AzHicham) [inline libs/mimir/src/adapters/primary/bragi/api.rs:256]: `Type::Zone | Type::City => Admin::static_doc_type(),`

File: libs/mimir/src/adapters/primary/bragi/handlers.rs [hunk 2]
```diff
@@ -205,6 +205,7 @@ fn build_es_indices_to_search(query: &ForwardGeocoderQuery) -> Vec<String> {
                 Type::House => indices.push(root_doctype(Addr::static_doc_type())),
                 Type::Street => indices.push(root_doctype(Street::static_doc_type())),
                 Type::Zone => indices.push(root_doctype(Admin::static_doc_type())),
+                Type::City => indices.push(root_doctype(Admin::static_doc_type())),
```
Comment (AzHicham) [inline libs/mimir/src/adapters/primary/bragi/handlers.rs:208]: `Type::Zone | Type::City => Admin::static_doc_type(),`

**Review by AzHicham — APPROVED** [review]

File: libs/mimir/src/adapters/primary/bragi/handlers.rs (hunk 2 above)
Comment (azime) [reply libs/mimir/src/adapters/primary/bragi/handlers.rs]: :+1:

File: libs/mimir/src/adapters/primary/bragi/api.rs (hunk 1 above)
Comment (azime) [reply libs/mimir/src/adapters/primary/bragi/api.rs]: :+1:

**Pull request merged**